data/resume/jbatorski.json

```diff
@@ -92,7 +98,13 @@
     {
       "name": "Answers Media",
       "location": "Chicago, IL",
-      "position": "Software Developer",
+      "positions": [
```

Comment:
Modified this to not deal with two different ways of representing position data and dealing with null checks in the react components. This does duplicate the start/end dates, which could be derived in the root work item object. Starting to deviate more and more from the resume schema definition, will see how this goes or if I decide to backtrack and stick to the schema definition more and create a transformation function to convert the raw data into something that matches up with what the UI needs more.

Reply:
Created this tracker item (https://github.com/jarekb84/me/issues/6) to evaluate if these extensions could be reverted, and added back via transformation function to provide a DS that better fits the UI needs.